src/content/en/updates/2015/01/ES6-Template-Strings.md

```diff
@@ -80,7 +80,7 @@ If you require backticks inside of your string, it can be escaped using the back
 
 ## Multiline Strings
 
-Multiline strings in JavaScript have required hacky workarounds for some time. Current solutions for them require that strings either exist on a single line or be split into multiline strings using a `\` (blackslash) before each newline. For example:
+Multiline strings in JavaScript have required hacky workarounds for some time. Current solutions for them require that strings either exist on a single line or be split into multiline strings using a `\ ` (blackslash) before each newline. For example:
```

Comment:
There was a typo in the original sentence: blackslash should be backslash.

Also, you should fix this backslash rendering problem on line `77` as well.

Finally, are you sure this shouldn't be fixed with `` `\\` `` instead?

Reply:
The internal devsite markdown guide has a bit more detail.  It says:

In an inline code span, spaces immediately following the opening backtick as
well as those immediately preceding the closing backtick are not rendered. So if
an inline code span ends with a backslash (`\ `), add a space before the closing
backtick to render a literal backslash.

Reply:
Updated with the spelling fix, a date update, and the second backslash.